frappe/desk/doctype/todo/todo.py

```diff
@@ -97,9 +99,22 @@ def has_permission(doc, user):
 	else:
 		return doc.owner==user or doc.assigned_by==user
 
+def update_overdue():
+	today = nowdate()
+	todos = frappe.db.get_all('ToDo', filters={'status': 'Open', 'date': ['<', today]}, fields=['name', 'date'])
+
+	for todo in todos:
+		if todo.get('date'):
+			if get_datetime(today) >= get_datetime(todo.get('date')):
+				doc = frappe.get_doc('ToDo', todo.get('name'))
+				doc.status = 'Overdue'
+				doc.save()
```

Comment:
this will update the last modified by and make it "Administrator", use `frappe.db.set_value`

Reply:
@rmehta Thank you, fixed.